### ISSUE TITLE
VB -> C#: Weird whitespace for comments

### ISSUE THREAD

**Issue by mrmonday** [icsharpcode/CodeConverter#334]
### Input code
```vbnet
Class Foo

End Class

''' <summary>
''' Something
''' </summary>
Friend Class Program
    Public Shared Sub Main(ByVal args As String())
    End Sub
End Class
```

### Erroneous output
```c#
namespace ConsoleApp4
{
    class Foo
    {
    }

    /// <summary>

/// Something

/// </summary>
    internal class Program
    {
        public static void Main(string[] args)
        {
        }
    }
}
```

### Expected output
```c#
namespace ConsoleApp4
{
    class Foo
    {
    }

    /// <summary>
    /// Something
    /// </summary>
    internal class Program
    {
        public static void Main(string[] args)
        {
        }
    }
}

```

### Details
Product in use: VS extension

Version in use: 6.9.0.0

**Comment by GrahamTheCoder** [comment]
We should make sure this is covered in the project and solution tests when it's fixed. Though previously I found a similar issue caused by newline differences, which makes it annoying to test because git gets involved

**Comment by Saibamen** [comment]
Should be fixed in release 8.0.2: https://github.com/icsharpcode/CodeConverter/releases/tag/8.0.2
Please retest
